charts/karpenter/Chart.yaml

```diff
@@ -1,6 +1,14 @@
 apiVersion: v2
-appVersion: 0.5.4
 name: karpenter
-description: A Helm chart for https://github.com/aws/karpenter/.
+description: A Helm chart for Karpenter, an open-source node provisioning project built for Kubernetes.
 type: application
 version: 0.5.4
+appVersion: 0.5.4
+keywords:
+  - fluent
+  - fluentd
+  - logging
```

Comment:
Probably a typo and we want something like autoscaling and karpenter here :thinking:

Reply:
@mbevc1 I completely forgot to put in the correct keywords. These are from the chart I used as a template. My suggestion would be as follows.

- cluster
- node
- scheduler
- autoscaling
- lifecycle